package.json

```diff
@@ -91,8 +91,8 @@
     "linux-blockutils": "0.1.0",
     "lodash": "4.17.15",
     "mime": "2.4.4",
-    "minimist": "1.2.0",
-    "mkdirp": "0.5.1",
+    "minimist": "1.2.5",
+    "mkdirp": "1.0.4",
```

Comment:
mkdirp major version changed from callback to promise, no need for code changes too??
See here - https://github.com/isaacs/node-mkdirp/blob/master/CHANGELOG.md